content/en/docs/setup/independent/troubleshooting-kubeadm.md

```diff
@@ -159,7 +159,7 @@ sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
 sudo chown $(id -u):$(id -g) $HOME/.kube/config
 ```
 
-### Default NIC When using flannel as the pod network in Vagrant
+### Default NIC When using flannel as the pod network in Vagrant or if you have VirtualBox that use an only-host network interface
```

Comment:
It might be better to generalize this section a bit more. The real issue is flannel choosing the wrong network interface to use when multiple are set. While commonly this is when using Vagrant w/ VirtualBox, there are other scenarios that could lead to it as well.